etc/kayobe/environments/ci-multinode/kolla.yml

```diff
@@ -2,3 +2,5 @@
 kolla_enable_cinder: true
 kolla_enable_cinder_backup: true
 kolla_enable_neutron_provider_networks: true
+kolla_enable_ovn: true
+kolla_enable_octavia: true
```

Comment:
Perhaps only enable octavia when OVN is enabled, since we don't have the config for the amphora provider in place currently.